_episodes/03-skimage-images.md

```diff
@@ -101,6 +101,54 @@ image = skimage.io.imread(fname="chair.jpg")
 ~~~
 {: .python}
 
+> ## Import Statements in Python
+>
+> In Python, the `import` statement is used to load additional functionality
+> into a program. This is necessary when we want our code to do something more
+> specialised, which cannot easily be achieved with the limited set of basic
+> tools and data structures available in the default Python environment.
+>
+> Additional functionality can be loaded as a single function or object,
+> a module defining several of these, or a library containing many modules.
+> You will encounter several different forms of `import` statement.
+>
+>
+> ~~~
+> import skimage                 # form 1, load whole skimage library
+> import skimage.io              # form 2, load skimage.io module only
+> from skimage.io import imread  # form 3, load only the imread function
+> import numpy as np             # form 4, load all of numpy into an object called np
+> ~~~
+> {: .language-python }
+>
+> > ## Further Explanation
+> >
+> > In the example above, form 1 loads the entire `skimage` library into the
+> > program as an object. individual modules of the library are then available
+> > within that object, e.g. to access the `imread` function used
+> > in the example above, you would write `skimage.io.imread()`.
+> >
+> > Form 2 loads only the `io` module of `skimage` into the program. When we run
+> > the code, the program will take less time and use less memory because we will
+> > not load the whole `skimage` library. The syntax needed to use the module
+> > remains unchanged.: to access the `imread` function, we would use the same
+> > function call as given for form 1.
+> >
+> > To further reduce the time and memory requirements for your program,
+> > form 3 can be used to import only a specific function from a library/module.
+> > In the example above, the `imread` function could then be called without
+> > the need for the preceding library and function names, i.e. `imread()`.
+> >
+> > Finally, the `as` keyword can be used when importing, to define a name to be
+> > used as shorthand for the library/module being imported. You may see `as`
+> > combined with any of the other first three forms of `import` statement.
+> >
+> > Which form is used often depends on the size and number of additional tools
+> > being loaded into the program.
+> >
+> {: .solution }
+{: .callout }
+
 First, we import the `io` module of skimage (`skimage.io`) so
 we can read and write images. Then, we use the `skimage.io.imread()` function to read
 a JPEG image entitled **chair.jpg**. Skimage reads the image, converts it from
```

Comment:
Move the callout below this paragraph, so the paragraph stays closer to the code block it refers to? (on a small screen, people might have to scroll back to see the code block).

Reply:
Great point, thanks. Fixed in b83defb.